README.md

```diff
@@ -2,6 +2,67 @@
 
 These are the scripts I use while releasing [The Foreman](https://theforeman.org). They automate most of the [release process](https://github.com/theforeman/tool_belt/blob/master/procedures/foreman/release.md.erb).
 
+## Setup
+
+Required dependencies:
+
+ * gopass
+ * python3
+  * click
+  * python-jenkins
+
+### Gopass
+
+First, clone this repository to your local machine. [gopass](https://github.com/gopasspw/gopass) will need to be installed and then set up on your machine for the release environment.
+
+Ensure that `gopass` is initialized after installing the first time (and that your GPG private key is present on the system):
+
+```
+gopass init
+```
+
+After installing `gopass`, the shared password stores need to be cloned. This requires setting up SSH configuration to the Foreman puppetmaster:
+
+Add to ~/.ssh/config:
+
+```
+Host foreman-pm
```

Comment:
I prefer using `puppetmaster.theforeman.org` as the `Host` and not set `HostName`, but that might be me and should not block this PR.

Reply:
Hah, well I copied this from the wiki!